renderer/brave_content_renderer_client.cc

```diff
@@ -139,6 +140,10 @@ void BraveContentRendererClient::RenderFrameCreated(
         base::BindRepeating(&BraveRenderThreadObserver::GetDynamicParams));
   }
 
+  // Used to inject scripts into the page.
+  // Currently only used by PSST feature.
```

Comment:
this extra comment line is not necessary. In fact I'd say the entire comment is not necessary if you have add the comments I suggested to ScriptInjectorRenderFrameObserver header file

Reply:
It looked like all the other render frame observers here were behind a feature flag, so just wanted to clarify that why this one wasn't. But if it's going to become redundant v soon, there's no point.

Reply:
it's just coincidence that all the others are behind feature flags. That isn't the case in ChromeContentRendererClient